pkg/config/template.go

```diff
@@ -1271,6 +1272,7 @@ const templateStringCrioRuntimeRuntimesRuntimeHandler = `# The "crio.runtime.run
 # - no_sync_log (optional, bool): If set to true, the runtime will not sync the log file on rotate or container exit.
 #   This option is only valid for the 'oci' runtime type. Setting this option to true can cause data loss, e.g.
 #   when a machine crash happens.
+# - default_annotations (optional, map): Default annotations if not overridden by the pod spec.
```

Comment:
You might need to add [this](https://github.com/cri-o/cri-o/pull/7180/files#diff-22a2b88ec18d80ceabdae558506a636efbaf4d9008ba36bfe93db19d8f0be113R1258) for default_annotations.